config/configrotate/configrotate.go

```diff
@@ -0,0 +1,54 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package configrotate // import "go.opentelemetry.io/collector/config/configrotate"
+
+import (
+	"io"
+	"os"
+
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+type Config struct {
+	// Enabled controls whether or not rotate logs
+	Enabled bool `mapstructure:"enable"`
+
+	// MaxMegabytes is the maximum size in megabytes of the file before it gets
+	// rotated. It defaults to 100 megabytes.
+	MaxMegabytes int `mapstructure:"max_megabytes"`
```

Comment:
```suggestion
	MaxMb int `mapstructure:"max_mb"`
```

Reply:
Please add tests to check behavior if set to negative values. Same for max days.

Reply:
The config was copied from [fileexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/fileexporter). I think we'd better use the same config for similar functions.